From 2fa32614480c36e53fdfbb583a45958eb0548e85 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 1 Sep 2011 16:45:21 +0800
Subject: [PATCH 0393/2463] ENGR00155761 usb-host: fix the buiild warning with
 upstream reviewing patch

Warning message:
/home/b29397/work/projects/linux-2.6-imx/drivers/usb/host/ehci.h:
748: warning: function declaration isn't a prototype

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 drivers/usb/host/ehci.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/host/ehci.h b/drivers/usb/host/ehci.h
index 1a35675..c208d0b 100644
--- a/drivers/usb/host/ehci.h
+++ b/drivers/usb/host/ehci.h
@@ -776,12 +776,12 @@ static inline unsigned ehci_read_frame_index(struct ehci_hcd *ehci)
  * descriptor to memory.
  * */
 #ifdef	CONFIG_ARM_DMA_MEM_BUFFERABLE
-static inline void ehci_sync_mem()
+static inline void ehci_sync_mem(void)
 {
 	mb();
 }
 #else
-static inline void ehci_sync_mem()
+static inline void ehci_sync_mem(void)
 {
 }
 #endif
-- 
1.7.7.4

